lib/stripe/credit_card_clone_finder.rb

```diff
@@ -14,10 +14,10 @@ def find_cloned_card
       customers = Stripe::Customer.list({ email: email, limit: 100 },
                                         stripe_account: @stripe_account)
 
-      customers.auto_paging_each do |customer|
+      customers.data.each do |customer|
```

Comment:
This should be okay most of the time but data may accumulate over time and one day this may break. Looking into the migration guides, release notes and finally in the Stripe source code it looks like this method was removed accidentally. I opened an issue:

* https://github.com/stripe/stripe-ruby/issues/1110

Let's see what they reply. Maybe we can keep the old code. Otherwise I would be inclined to implement our own version of `auto_paging_each`.